astro.config.ts

```
@@ -1,58 +1,47 @@
 import { defineConfig } from "astro/config";
-import tailwind from "@astrojs/tailwind";
-import react from "@astrojs/react";
+import tailwindcss from "@tailwindcss/vite";
+import sitemap from "@astrojs/sitemap";
 import remarkToc from "remark-toc";
 import remarkCollapse from "remark-collapse";
-import sitemap from "@astrojs/sitemap";
 import { SITE } from "./src/config";
+
+import react from "@astrojs/react";
 import rehypeMermaid from "rehype-mermaid";
```

ESLint

[error] 'rehypeMermaid' is defined but never used.

(@typescript-eslint/no-unused-vars)